client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEditStepForm.jsx

```diff
@@ -608,6 +609,8 @@ const SubmissionEditStepForm = (props) => {
             {question.type === questionTypes.Programming &&
               isCodaveriEnabled &&
               question.isCodaveri &&
+              question.liveFeedbackEnabled &&
+              assessment.liveFeedbackEnabled &&
```

Comment:
do we need to check `course.liveFeedbackEnabled` here?

Reply:
No, because when [the backend state is populated](https://github.com/Coursemology/coursemology2/pull/7433/files#diff-3935da25a4fcfc1d8a7b0758d7de23ee292931b129bba299ced381f73e358f27R28), it already adjusts that field if live feedback is disabled on the course level.